nikola/nikola.py

```diff
@@ -1031,7 +1031,7 @@ def init_plugins(self, commands_only=False, load_all=False):
         extra_plugins_dirs = self.config['EXTRA_PLUGINS_DIRS']
         self._loading_commands_only = commands_only
         self._plugin_places = [
-            resource_filename('nikola', 'plugins'),
+            str(resources.path('nikola', 'plugins')) if sys.version_info.minor == 8 else str(resources.files('nikola').joinpath('plugins')),
```

Comment:
1. Instead of repeating the same version check everywhere, please instead make a helper function in `utils.py`.
2. Please make it so your version check won’t break for Python 4.8, and consider making it `<=` instead of `==` while doing that.
3. What do the functions return so that they need `str`? If it’s a `pathlib.Path`, it would be desirable to switch other things to `Path`s instead of working with strings.